### PR TITLE
[feat] added cart and wishlist functionality

### PR DESCRIPTION
**Pull request by saurabhan**
added cart and wishlist functionality

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_CHEC_SECRET = pk_41913b5d7ff91d787962e562127f9a3b32763054a5852
```
Comment (itsmebipul00) [inline frontend/.env:1]: you could gitignore this file

File: frontend/src/pages/Products.jsx [hunk 2]
```diff
@@ -66,39 +33,36 @@ function Products() {
...
-                        ))}
-                      </div>
-                    </fieldset>
-                  </div>
-                ))}
-              </form>
+              <h1 className="font-bold">Price</h1>
+              <div className="flex flex-col">
+
+              <label class="inline-flex items-center mt-3">
```
Comment (itsmebipul00) [inline frontend/src/pages/Products.jsx:39]: You could use for in label and id in input

File: frontend/src/pages/Wishlist.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  const { wishlist, removeFromWishlist } = useWishlist()
+  return (
+      <div>
+          <Navbar/>
+      <main className="max-w-2xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:max-w-7xl lg:px-8">
+          <h1 className="text-3xl font-bold">Wishlist</h1>
+      <div className="mt-6 lg:mt-0 lg:col-span-2 xl:col-span-3">
+
+      <div className="mt-8 grid grid-cols-1 gap-y-12 sm:grid-cols-2 sm:gap-x-6 lg:grid-cols-4 xl:gap-x-8">
+          {wishlist.map((product) => (
```
Comment (itsmebipul00) [inline frontend/src/pages/Wishlist.jsx:18]: You can reuse your product card component here

**Review by itsmebipul00 — APPROVED** [review]
Awesome UI, good to merge

File: frontend/src/context/WishlistContext.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+const WishlistContext = createContext()
+const useWishlist = () => useContext(WishlistContext)
+
+const WishlistProvider = ({children}) =>{
+
+    const [ wishlist, setWishlist ] = useState([]);
+
+
+    const addToWishlist = (product, id) =>{
+        console.log('addedtowishlist', product)
```
Comment (Ranjeetk489) [inline frontend/src/context/WishlistContext.jsx:13]: remove console.log statement

File: frontend/src/pages/Wishlist.jsx (hunk 3 above)
Comment (Ranjeetk489) [reply frontend/src/pages/Wishlist.jsx:18]: +1

**Review by Ranjeetk489 — APPROVED** [review]
Loved the clean and appeasing UI
good to merge

**Pull request merged**